process/block/preprocess/gasComputation_test.go

```diff
@@ -23,22 +22,37 @@ func TestNewGasComputation_NilEconomicsFeeHandlerShouldErr(t *testing.T) {
 	gc, err := preprocess.NewGasComputation(
 		nil,
 		&testscommon.TxTypeHandlerMock{},
-		&epochNotifier.EpochNotifierStub{},
-		0,
+		&testscommon.EnableEpochsHandlerStub{
```

Comment:
could have declared a function that returns the enable epochs handler with that setting on